theme-utils.mjs

```diff
@@ -646,6 +706,50 @@ async function pushChangesToSandbox() {
 	}
 }
 
+async function pullCoreThemes() {
+	console.log("Pulling CORE themes from sandbox.");
+	for (let theme of coreThemes ) {
+		await executeCommand(`
+			rsync -avr --no-p --no-times --delete -m --exclude-from='.sandbox-ignore' wpcom-sandbox:${sandboxPublicThemesFolder}/${theme}/ ./${theme}/ 
+		`, true);
+	}
+}
+
+async function pushCoreThemes() {
+	console.log("Pushing CORE themes to sandbox.");
+	for (let theme of coreThemes ) {
+		await executeCommand(`
+			rsync -avr --no-p --no-times --delete -m --exclude-from='.sandbox-ignore' ./${theme}/ wpcom-sandbox:${sandboxPublicThemesFolder}/${theme}/ 
+		`, true);
+	}
+}
+
+async function syncCoreTheme(theme, sinceRevision) {
+	if(!theme){
+		console.log('Must supply theme to sync and revision to start from');
+		return;
+	}
+	if(!sinceRevision) {
+		sinceRevision = await executeOnSandbox(`cat ${sandboxPublicThemesFolder}/${theme}/.pub-svn-revision`);
+	}
+	let latestRevision = await executeCommand(`svn info -r HEAD https://core.svn.wordpress.org/trunk | grep Revision | egrep -o "[0-9]+"`);
```

Comment:
Thank you for working on this!!

To be sure, we want to use `core.svn` instead of `develop.svn`? I'm just wondering if there would be any benefit to syncing to the development version of Core, since it is often ahead of a release.  

For example, right now there are some changes to TT2 on trunk right now that would be good to sync back to wpcom, but would not be reflected if we point to `core.svn`.

Reply:
That's a decision I'm happy for !me to make. :)

I think that syncing from develop is likely more the exception.  I see it as a tool mostly to sync "unknown changes that have happened" than "specific work that we need in wpcom sooner-than-later".

Perhaps it should be an argument?

Reply:
Argument makes sense to me and could be addressed in a follow up :)